snapcraft/main.py

```diff
@@ -121,6 +122,7 @@
   status       Show the current status of a snap per channel and architecture.
   list-revisions List all revisions of a snap.
   revisions    Alias for list-revisions.
```

Comment:
Just to throw this out as a rhetorical comment: I dislike having two commands that do the same thing (`revisions` and `list-revisions`), it seems confusing and I'm not clear on the benefit.